flag.go

```diff
@@ -271,9 +271,18 @@ func stringifyFlag(f Flag) string {
 	needsPlaceholder := false
 	defaultValueString := ""
 	val := fv.FieldByName("Value")
+
+	showDefaultValue := true
+	if boolFlag, ok := f.(*BoolFlag); ok {
+		showDefaultValue = boolFlag.ShowDefaultValue
+	}
```

Comment:
Unrelated tests are failing because the default value for the flag `ShowDefaultValue` is `false`, despite you setting `showDefaultValue := true` here. Given that "show" needs to be the default behavior, and the default value for `bool` if `false`, I think you need to change the flag option from `BoolFlag.ShowDefaultValue` to `BoolFlag.HideDefaultValue` (and change the logic accordingly).

Reply:
pls check now